container_images/test-manager/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ValidateReturn() {
+  RET=$?
```

Comment:
i am not even sure this will work - you're in a new function!
but it is in any case an inappropriate use of bash functions. just move your print logic to where this was invoked

Reply:
Just echo execute return status where it was invoked.

Reply:
i didnt ask what it does. i said i don't think it will work as written, and that it's bad practice anyway.

Reply:
I mean I make the change (remove this function) to avoid this bad practice. but originally code did work:)